libs/wingsdk/src/target-tf-gcp/function.ts

```diff
@@ -12,14 +12,31 @@ import {
   NameOptions,
   ResourceNames,
 } from "../shared/resource-names";
-import { IInflightHost } from "../std";
+import { IInflightHost, IResource } from "../std";
 
 const FUNCTION_NAME_OPTS: NameOptions = {
   maxLen: 32,
   disallowedRegex: /[^a-z0-9]+/g,
   case: CaseConventions.LOWERCASE,
 };
 
+export enum ResourceTypes {
+  BUCKET = "Bucket",
+  FUNCTION = "Function",
+}
+
+export enum ActionTypes {
+  STORAGE_READ = "roles/storage.objectViewer",
+  STORAGE_READ_WRITE = "roles/storage.objectUser",
+  FUNCTION_INVOKER = "roles/cloudfunctions.invoker",
+  FUNCTION_VIEWER = "roles/cloudfunctions.viewer",
+}
```

Comment:
Can we link to any docs for these?

Reply:
I'm not sure about this as it is only GCP specific. Would love to know your thought on this

Reply:
I guess Chris meant to put the link to the gcp docs related to the permission type  as a comment next to each